plugins/plugin-chart-echarts/src/Graph/controlPanel.tsx

```diff
@@ -0,0 +1,170 @@
+  config: {
+    ...controlConfig,
+    clearable: false,
+    label: t('Target'),
+    description: t('Target for nodes of graph'),
+  },
+};
+
+const categoryControl = {
+  name: 'category',
+  config: {
+    ...controlConfig,
+    clearable: true,
+    label: t('Category'),
+    description: t('Optional category for nodes of graph'),
+    validators: [],
+  },
+};
+
+export default {
+  controlPanelSections: [
+    sections.legacyRegularTime,
+    {
+      label: t('Query'),
+      expanded: true,
+      controlSetRows: [
+        [sourceControl],
+        [targetControl],
+        [categoryControl],
+        ['metric'],
+        ['adhoc_filters'],
+        ['row_limit'],
+      ],
+    },
+    {
+      label: t('Chart Options'),
+      expanded: true,
+      controlSetRows: [
+        ['color_scheme', noopControl],
+        [
+          {
+            name: 'layout',
+            config: {
+              type: 'SelectControl',
+              renderTrigger: true,
+              label: t('Graph Layout'),
+              default: DEFAULT_FORM_DATA.layout,
+              choices: formatSelectOptions([
+                ['force', t('Force')],
+                ['circular', t('Circular')],
+              ]),
+              description: t('Layout type of graph'),
+            },
+          },
+        ],
+        [
+          {
+            name: 'draggable',
+            config: {
+              type: 'CheckboxControl',
+              label: t('Enable node draging'),
+              renderTrigger: true,
+              default: DEFAULT_FORM_DATA.draggable,
+              description: t('Whether to enable node dragging in force layout mode.'),
+            },
+          },
+          {
+            name: 'roam',
+            config: {
+              type: 'CheckboxControl',
+              label: t('Enable graph roaming'),
+              renderTrigger: true,
+              default: DEFAULT_FORM_DATA.roam,
+              description: t('Whether to enable chaging graph position.'),
+            },
+          },
+        ],
+        [
+          {
+            name: 'select_mode',
+            config: {
+              type: 'SelectControl',
+              renderTrigger: true,
+              label: t('Node Select Mode'),
+              default: DEFAULT_FORM_DATA.selectedMode,
+              choices: formatSelectOptions([
+                ['single', t('Single')],
+                ['multiple', t('Multiple')],
+              ]),
+              description: t('Allow node selections'),
+            },
+          },
+        ],
+        [
+          {
+            name: 'showSymbolThreshold',
+            config: {
+              type: 'TextControl',
+              label: t('Label Threshold'),
+              renderTrigger: true,
+              isInt: true,
+              default: DEFAULT_FORM_DATA.showSymbolThreshold,
+              description: t('Minimum value for label to be displayed on graph.'),
+            },
+          },
+        ],
+      ],
```

Comment:
It would be nice to add the default legend customization options that are now available in the other charts (enable/disable, orientation etc).

Reply:
legend is hidden by default,is it intended @villebro ?